src/utils/api.js

```diff
@@ -0,0 +1,92 @@
+import {
+  getFirestore,
+  collection,
+  addDoc,
+  doc,
+  deleteDoc,
+} from "firebase/firestore";
+import { firebaseConfig } from "./firebaseConfig";
+import firebase from "firebase/compat/app";
+import { getAuth } from "firebase/auth";
+
+console.log(firebase);
+
+const app = firebase.initializeApp(firebaseConfig);
+
+export const auth = getAuth(app);
```

Comment:
Don't think app is needed to be passed to the getAuth method: 

https://firebase.google.com/docs/auth/web/start